wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/CodenvyLicenseManager.java

```diff
@@ -151,34 +117,33 @@ private void removeActionsOfExpiredLicense() throws ServerException {
      */
     @Nullable
     public CodenvyLicense load() throws LicenseException {
-        String licenseText = readLicenseText();
-        if (isNullOrEmpty(licenseText)) {
-            throw new LicenseNotFoundException("Codenvy license not found");
-        }
-
-        return licenseFactory.create(licenseText);
+        String licenseText = codenvyLicenseStorage.loadLicense();
+        CodenvyLicense codenvyLicense = licenseFactory.create(licenseText);
+        codenvyLicenseActivator.validateActivation(codenvyLicense);
```

Comment:
Do we have a way to store the product license which doesn't require activation at all without exception?